e2e/challenge-description.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { test, expect, type Page } from '@playwright/test';
+
+let page: Page;
```

Comment:
```suggestion
import { test, expect } from '@playwright/test';
```

We don't need a global page variable because...

Reply:
Removed global page variable.